OpenOversight/app/__init__.py

```diff
@@ -95,6 +95,13 @@ def get_age_from_birth_year(birth_year):
         if birth_year:
             return int(datetime.datetime.now().year - birth_year)
 
+    @app.template_filter('currently_on_force')
+    def officer_currently_on_force(assignments):
+        if not assignments:
+            return "Uncertain"
+        most_recent = max(assignments, key=lambda x: x.star_date or datetime.date.min)
+        return "Yes" if most_recent.resign_date is None else "No"
+
```

Comment:
Is a filter the best way to do this? I'm totally open to it but it would make more sense to me to add a `@property` to the officer or something like that, if possible...

Reply:
Unfortunately this is tied to `Assignments` and not `Officers`, so it's one hop away and would require querying assignments from the officer model I think. It's possible! This just seemed a bit easier to throw in because `assignments` were already available for a given officer in this template

Reply:
Ah okay, that makes sense. Thanks for explaining the reasoning.